frontend/src/dto/users/users.res.dto.ts

```diff
@@ -1,7 +1,10 @@
 import { User } from "models/users.model";
+import { OneChonInfo } from "types/chon.types";
 
 export type GetUserResDto = User;
 
 export type PostSignInResDto = User;
 
 export type PostSignUpResDto = User;
+
+export type GetChonListResDto = OneChonInfo[];
```

Comment:
api response는 json 형태로 옵니다. Array로 올 수 없습니다. 

```suggestion
export interface GetChonListResDto {
  oneChonInfo: Array<OneChonInfo>
}
```